### DISCUSSION TITLE
AMA Questions Pre-Submission and Voting

### DISCUSSION THREAD

**Discussion by christophwille** [icsharpcode/ILSpy#2720] [General]
We will be doing a hybrid online/offline AMA on Thursday July 14th - https://www.meetup.com/dotnet-austria/events/286687693/

If you want to submit questions ahead of time, please use this discussion thread. DO vote (thumbs up) on questions submitted by others! We will rank questions by how popular it is and then get to them during the AMA.

**Comment by Flash0ver** [comment]
Tell me a little about the history of _ILSpy_.

**Comment by Flash0ver** [comment]
What was the toughest _C#_ language feature to implement, and which was the easiest?
